src/index.js

```diff
@@ -1,6 +1,8 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ]  linter is used to flag programming errors, bugs, stylistic errors, and suspicious constructs, so to keep your code readable and understandable by everyone, kindly remove the linters disabling command and please fix all the linters error 🙏